handleleave_test.go

```diff
@@ -221,3 +222,177 @@ func TestHandleMakeLeave(t *testing.T) {
 		})
 	}
 }
+
+type dummyQuerier struct {
+	pdu PDU
```

Comment:
It would be good to assert the type/state key are correct when queried via `CurrentStateEvent`.